python/res/job_queue/workflow.py

```diff
@@ -121,4 +121,8 @@ def createCReference(cls, c_pointer, parent=None):
         workflow.__current_job = None
         return workflow
 
+    def __ne__(self, other):
+        return not (self == other)
 
+    def __eq__(self, other):
+        return self.src_file == other.src_file
```

Comment:
I guess this comparison will fail if you haver relative paths, or if you are going through symlinks. Maybe use something like `os.path.realpath()`